src/live.rs

```diff
@@ -732,6 +732,11 @@ fn ignition_embed_area(iso: &mut IsoFs) -> Result<Region> {
 struct LiveIgnition {
     /// The Ignition config for the live system
     live: Ignition,
+    /// The Ignition config for the destination system
+    dest: Option<Ignition>,
+    /// A user-supplied Ignition config for the dest system, which might be
+    /// merged into the dest config or might become the dest config
+    deferred_dest: Option<ignition_config::Config>,
```

Comment:
Would it be cleaner and easier to understand to just keep it as a `Vec<ignition_config::Config>` and then we can decide based on `dest.len()` whether to wrap or not at `into_bytes()` time?